inference-engine/src/mkldnn_plugin/nodes/mkldnn_tile_node.h

```diff
@@ -4,31 +4,42 @@
 
 #pragma once
 
-#include <ie_common.h>
-#include <mkldnn_node.h>
+#include "common/tile_broadcast_utils.h"
+
 #include <string>
 
 namespace MKLDNNPlugin {
 
-class MKLDNNTileNode : public MKLDNNNode {
+class MKLDNNTileNode : public MKLDNNNode, public TileBroadcastCommon {
 public:
-    MKLDNNTileNode(const std::shared_ptr<ngraph::Node>& op, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache);
+    MKLDNNTileNode(const std::shared_ptr<ov::Node>& op, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache);
 
     void getSupportedDescriptors() override;
     void initSupportedPrimitiveDescriptors() override;
     void createPrimitive() override;
     void execute(mkldnn::stream strm) override;
+    void executeDynamicImpl(mkldnn::stream strm) override {
+        execute(strm);
+    }
+    void plainExecute(mkldnn::stream strm);
     bool created() const override;
 
-    static bool isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept;
+    static bool isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std::string& errorMessage) noexcept;
+
+protected:
+    bool needPrepareParams() const override;
+    void prepareParams() override;
+    bool needShapeInfer() const override;
+    std::vector<VectorDims> shapeInfer() const override;
 
 private:
-    static const size_t TILE_INPUT = 0;
-    static const size_t TILE_REPEATS = 1;
+    static constexpr size_t TILE_INPUT = 0lu;
+    static constexpr size_t TILE_REPEATS = 1lu;
 
     int axis = -1;
     int tiles = 0;
     bool noTiling = false;
+    mutable InferenceEngine::SizeVector originRepeats;
```

Comment:
This vector is not modified in constant method anymore, so the `mutable` qualifier can be removed.

Reply:
Done.